spec/models/user_spec.rb

```diff
@@ -11,8 +11,8 @@
     it { is_expected.to have_many(:needs).dependent(:destroy) }
     it { is_expected.to have_many(:notes).dependent(:destroy) }
     it { is_expected.to have_many(:assigned_contacts).through(:needs).source(:contact) }
-    it { is_expected.to have_many(:uncompleted_needs).conditions(completed_on: nil) }
-    it { is_expected.to have_many(:completed_needs).conditions('completed_on IS NOT NULL') }
+    it { is_expected.to have_many(:uncompleted_needs).conditions('status <> complete') }
```

Comment:
see comment above - I think this is correct - above needs updating.

Reply:
Not sure what you mean. Here we are testing needs sopes completed and uncompleted.

Reply:
Not sure what you mean. Here we are testing needs sopes completed and uncompleted.